src/main/resources/default/taglib/w/tableRow.html.pasta

```diff
@@ -0,0 +1,47 @@
+<i:arg name="titleKey" type="String" default=""/>
```

Comment:
this template seems oddly specific... and also there is infobox and infoboxElement

Reply:
you are right, infobox is good 😋